podfile

```diff
@@ -1,4 +1,4 @@
 platform :ios, '11.0'
 target 'ArcGIS Runtime SDK Samples' do
-  pod 'ArcGIS-Runtime-SDK-iOS', '~> 100.6'
+  pod 'ArcGIS-Runtime-Toolkit-iOS', '~> 100.5'
```

Comment:
This will need to be updated to 100.6 once the Toolkit is released.

Reply:
Yup

Reply:
And also the submodule reference